Backend/auth-utils/hashPassword.js

```
@@ -0,0 +1,12 @@
+const bcrypt = require("bcryptjs");
+
+function hashPassword(originalPass, hashingMethod) {
+  bcrypt.genSalt(10, (error, salt) => {
+    if (error) {
+      console.log(error.message);
```

Error was not handled properly.

Recommendation:
Return the error data on your `hashingMethod` callback and handle it accordingly.